runner/src/builtins/llm.ts

```diff
@@ -16,15 +19,16 @@ import { type ReactivityLog } from "../scheduler.ts";
  * Returns the complete result as `result` and the incremental result as
  * `partial`. `pending` is true while a request is pending.
  *
- * @param prompt - A doc to store the prompt message - if you only have a single message
- * @param messages - list of messages to send to the LLM. - alternating user and assistant messages.
+ * @param prompt - The prompt message to send to the LLM - if you only have a single message
+ * @param messages - List of messages to send to the LLM - alternating user and assistant messages.
  *  - if you end with an assistant message, the LLM will continue from there.
  *  - if both prompt and messages are empty, no LLM call will be made,
  *    result and partial will be undefined.
- * @param model - A doc to store the model to use.
- * @param system - A doc to store the system message.
- * @param stop - A doc to store (optional) stop sequence.
- * @param max_tokens - A doc to store the maximum number of tokens to generate.
+ * @param model - The LLM model to use
+ * @param system - The system message to set context for the LLM
+ * @param stop - Optional sequence that will stop generation when encountered
+ * @param max_tokens - Maximum number of tokens to generate
```

Comment:
Do we need to update this still?

https://github.com/commontoolsinc/labs/blob/main/builder/src/built-in.ts#L4-L20

Reply:
Yes! Added!